src/agent/onefuzz-task/src/tasks/task_logger.rs

```diff
@@ -440,7 +440,10 @@ pub struct SpawnedLogger {
 
 impl SpawnedLogger {
     pub async fn flush_and_stop(self, timeout: Duration) -> Result<()> {
-        let _ = tokio::time::timeout(timeout, self.logger_handle).await;
+        if let Ok(Err(e)) = tokio::time::timeout(timeout, self.logger_handle).await {
+            error!("task logger failure {}", e);
```

Comment:
NIT: maybe choose some different text to log so that when we see it it'll be easy to know exactly where it's coming from?

ex:
```suggestion
            error!("failed to flush and stop task logger {}", e);
```